src/dir.proj

```diff
@@ -0,0 +1,292 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <PropertyGroup>
+    <BuildTargets>
+      BuildProjectsForNuGetPackages;
+      PublishSharedFrameworkAndSharedHost;
+      GenerateRuntimeGraph;
+      CopyHostArtifactsToSharedFramework;
+      CrossGenDirectory;
+      GenerateVersionFile;
+    </BuildTargets>
+  </PropertyGroup>
+
+  <Target Name="Build" DependsOnTargets="$(BuildTargets)" />
+
+  <UsingTask TaskName="GetBuildArgsByFrameworks" AssemblyFile="$(BuildToolsTaskDir)core-setup.tasks.dll"/>
+
+  <Target Name="BuildProjectsForNuGetPackages">
+    
+    <PropertyGroup>
+      <pOutDir>$(IntermediateOutputRootPath)forPackaging</pOutDir>
```

Comment:
Can we move this into a shared property since it is defined in 2 spots?  (here and when we pack the nupkgs)